concepts/concept_rn_relatedrn_element.adoc

```diff
@@ -22,6 +22,7 @@ NOTE: You will be prompted to log in using your NetApp Support credentials.
 * https://library.netapp.com/ecm/ecm_download_file/ECMLP2873789[NetApp Element Software 12.2 Release Notes^]
 * https://library.netapp.com/ecm/ecm_download_file/ECMLP2877208[NetApp Element Software 12.0.1 Release Notes^]
 * https://library.netapp.com/ecm/ecm_download_file/ECMLP2865022[NetApp Element Software 12.0 Release Notes^]
+* https://library.netapp.com/ecm/ecm_download_file/ECMLP2880321[NetApp Element Software 11.8.2 Release Notes^] _NEW_
```

Comment:
Hey @netapp-yvonneo this ECM link points to a notice file for me.  Can you verify?  Maybe it's the wrong link?